config/application.rb

```diff
@@ -28,6 +28,8 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
+    require 'view_component'
+    require 'primer/view_components/engine'
```

Comment:
> In config/application.rb, add after the application definition:

https://primer.style/view-components/#installation

I believe that is what the docs specify.

Reply:
Do they mean _after_ as in after the `end`, or after the `do`?

Reply:
(As in, I think they might mean at the bottom of the file.)

Reply:
Now I'm not sure because the demo app in their repo place these requires after their railties. 😆 

https://github.com/primer/view_components/blob/main/demo/config/application.rb#L9-L11

I'll move them to match the demo app and confirm that works.